lib/fb/page.rb

```diff
@@ -39,5 +53,17 @@ def posts
     def to_s
       %Q(#<#{self.class.name} #{@id} "#{@name}">)
     end
+
+  private
+
+    def format(results)
```

Comment:
`format` and `format_metric` should problem go in another class since other resources (e.g. videos) will use it too. It will here for now. The next resources that require insights are posts which is still part of `Page`.